<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="LiftingArea">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Matrix Lifting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="matrixLiftingOutput"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="MultiArea">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Matrix Multiplication</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="matrixMultiplicationOutput"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="MinRgb">
           <property name="text">
            <string>Min RGB values(0 min)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MinRgbInput"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="MaxRgb">
           <property name="text">
            <string>Max RGB values (255 max)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MaxRgbInput"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Run Demo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
